docs/streams/kafka_on_gdn/kafka-python-client.md

```diff
@@ -0,0 +1,75 @@
+---
+sidebar_position: 20
+title: Kafka Python Client
+---
+
+Kafka-Python client library is designed to function much like the official java client, with a sprinkling of pythonic interfaces(e.g., consumer iterators). The library consists 6 different APIS via classes KafkaConsumer, KafkaProducer, KafkaAdminClient, KafkaClient, BrokerConnection, ClusterMetadata.
+
+The KafkaProducer can be used across threads without issue, unlike the KafkaConsumer which cannot. While it is possible to use the KafkaConsumer in a thread-local manner, multiprocessing is recommended.
```

Comment:
This paragraph can be rewritten into a note about KafkaConsumer limitations.

:::note
We recommend using KafkaConsumer with multiprocessing rather than thread-local.
:::

(You might have to fix that note, I do not know what the things you are talking about are exactly.)